### PR TITLE
New exploding animation

### PR DESCRIPTION
**Pull request by cecileboucheron**
This replaces the current moving squares with an explosion Lottie animation.
@buoyad I'll let you review.
cc @adamjspooner 

exploding-animation_1

exploding-image

exploding-capture

### CODE REVIEW

File: shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ class ExplodingHeightRetainer extends React.PureComponent<Props, State> {
           // to make sure we don't rewrap text when showing the animation
           this.props.retainHeight && {
             height: this.state.height,
-            overflow: 'hidden',
+            overflow: 'visible',
```
Comment (buoyad) [inline shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx:87]: let's remove this style key

File: shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx [hunk 2]
```diff
@@ -129,16 +129,22 @@ const Ashes = (props: {doneExploding: boolean; exploded: boolean; explodedBy?: s
...
 }
 
-const maxFlameWidth = 10
-const flameOffset = 5
 const FlameFront = (props: {height: number; stop: boolean}) => {
   if (props.stop) {
     return null
   }
-  const numBoxes = Math.ceil(props.height / 15)
+  const numBoxes = Math.ceil(props.height / 17 - 1)
```
Comment (buoyad) [inline shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx:140]: move `- 1` outside of `ceil`

File: shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ const copyChildren = (children: React.ReactNode): React.ReactNode =>
   // @ts-ignore
   React.Children.map(children, child => (child ? React.cloneElement(child) : child))
 
-export const animationDuration = 1500
+export const animationDuration = 3000
```
Comment (buoyad) [inline shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx:13]: FWIW I liked the original duration, this feels long to me

File: shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx (hunk 2 above)
Comment (buoyad) [reply shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx:140]: We should also make sure this is always at least 1

File: shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx (hunk 3 above)
Comment (cecileboucheron) [reply shared/chat/conversation/messages/wrapper/exploding-height-retainer/index.desktop.tsx]: 👍 moved it to 2000.

**Pull request merged**